### PR TITLE
Updated Terms of Service

### PR DESCRIPTION
**Pull request by scottharwell**
Updated Red Hat and IBM terms of service from 06/2023 changes.

### CODE REVIEW

File: docs/tos/index.md [hunk 1]
```diff
@@ -1,25 +1,137 @@
...
+Suggestions by ECS:  Based on your request, the ECS will provide suggested Ansible tasks and other Ansible materials (“Suggestions”). By using the ECS, you agree that any and all Suggestions and any User data input into ECS may be used by International Business Machines Corporation (“IBM”) and its affiliates including Red Hat, Inc. (“Red Hat”), for any purpose without limitation, including to further improve and train the ECS.
```
Comment (robinbobbitt) [inline docs/tos/index.md:13]: Do we want to underline with `<u>` to match the provided doc?

File: docs/tos/index.md [hunk 2]
```diff
@@ -1,25 +1,137 @@
...
-March, 2023
+### 2. Right to Use and User Responsibilities
+
+To use the ECS you need to install the Ansible Visual Studio Code extension and authenticate via your GitHub account that you have been authorized to use with the Ansible Service.  
+
+IBM is neither a party to the Red Hat Enterprise Agreement nor the Visual Studio Code agreement and is not responsible for the Ansible Service. User is responsible for the use of the ECS by any user who accesses the ECS with User's account credentials. User is responsible for obtaining all necessary rights and permissions to permit processing of data in the ECS.  
+
+With respect to Code, User shall apply meaningful testing and human oversight of the Code the User produces while using ECS to find and mitigate errors and vulnerabilities.
+
+The ECS is not designed for use in a production environment or for commercial purposes and any such use is at User’s own risk. If the offering becomes generally available, IBM is under no obligation to offer migration capabilities or services.
```
Comment (robinbobbitt) [inline docs/tos/index.md:31]: The first sentence of this paragraph is bold in the provided doc.

File: docs/tos/index.md [hunk 3]
```diff
@@ -1,25 +1,137 @@
...
+
+IBM does not undertake to perform any of User's regulatory obligations or assume any responsibility for User's business or operations, and User is responsible for its use of the ECS. IBM is acting as an information technology provider only. IBM's direction, suggested usage, or guidance or use of the ECS does not constitute medical, clinical, legal, accounting, or other licensed professional advice. User should obtain their own expert advice.
+
+Both parties agree to the application of the laws of the State of New York, United States, without regard to conflict of law principles.
+
+## Red Hat Terms of Service
+
+**Red Hat Ansible Lightspeed with IBM Watson Code Assistant Technical Preview Program Terms**
+
+We are excited that you are interested in participating in this Technical Preview Program (**“Program”**) and as a benefit, receiving access to Red Hat’s preview service that provides guidance on Ansible tasks and other Ansible related content (**“Lightspeed”**) integrated with IBM’s Early Cloud Service for Watson Code Assistant ("ECS") (subject to a separate IBM agreement). The Program gives you access to a preview version of Lightspeed at no cost subject to these terms.
```
Comment (robinbobbitt) [inline docs/tos/index.md:111]: "ECS" should be bold

File: docs/tos/index.md [hunk 4]
```diff
@@ -1,25 +1,137 @@
...
+
+The Program gives access to a preview version of Lightspeed. Lightspeed passes your requests to ECS and uses output from ECS to refine Ansible tasks and other Ansible related content. Red Hat shares that data and related information regarding such usage with IBM to train and improve Lightspeed and ECS. Lightspeed is not supported for or intended for any other purpose.
+
+To use Lightspeed, you need to install the Ansible Visual Studio Code extension and authenticate via your GitHub account that you have been authorized to use for this Program. Based on your request, Lightspeed will provide suggested tasks or other Ansible related materials (**"Suggestions"**). By participating in this Program you agree that any and all Suggestions and any data associated with your use, may be used by Red Hat and Red Hat’s affiliated companies, including IBM, for any purpose including to further improve and train Lightspeed and ECS. The following information is collected and may be shared with IBM:
+
+The VS Code extension provides prompts and context to create Suggestions, and user interactions with Suggestions such as acceptance, modification or rejection.
```
Comment (robinbobbitt) [inline docs/tos/index.md:119]: These sentences are bulleted in the provided doc.

**Review by robinbobbitt — CHANGES_REQUESTED** [review]

**Comment by scottharwell** [comment]
@robinbobbitt would you mind reviewing again.  Changes should be made.

**Review by robinbobbitt — APPROVED** [review]

**Pull request merged**